web/beacon-app/cypress/e2e/ProjectDetailPage.feature

```diff
@@ -0,0 +1,5 @@
+describe('template spec', () => {
+  it('passes', () => {
+    cy.visit('https://example.cypress.io')
+  })
+})
```

Comment:
i guess you forget to remove this ?

Reply:
I noticed I forgot to do this after creating the PR. This will be removed when the e2e test is implemented for the Project Detail page.